exchangelib/changes.py

```diff
@@ -26,6 +26,10 @@ class UpdateItemChange(ItemChange):
 
 class DeleteItemChange(ItemChange):
     ELEMENT_NAME = 'Delete'
+    FIELDS = [
+        IdAndChangekeyField('item_id', field_uri='ItemId'),
```

Comment:
What is `AndChangekeyField` referring to?

Reply:
So you know how in our code we have this `EWSItemId` named tuple that includes both an `id` and a `change_key`? The `IdAndChangekeyField` is basically just that, but un-named.